### PR TITLE
chore: add display name to chest prefab

### PR DESCRIPTION
**Pull request by jdrueckert**
Minor adjustment to add a display name for chests.
Especially the feature added in https://github.com/Terasology/Inventory/pull/45 will benefit of this.

### CODE REVIEW

**Comment by jdrueckert** [comment]
[Improve usability of inventory screens](https://trello.com/c/zM9gef21/152-improve-usability-of-inventory-screens)

File: assets/prefabs/furnishings/chest.prefab [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+    "DisplayName": {
+        "name": "Chest"
```
Comment (skaldarnar) [inline assets/prefabs/furnishings/chest.prefab]: Can we make this translatable?

**Review by skaldarnar — APPROVED** [review]
We can make it translatable :upside_down_face:

File: assets/prefabs/furnishings/chest.prefab [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+    "DisplayName": {
+        "name": "${coreadvancedassets:game#furnishings-chest}"
```
Comment (skaldarnar) [inline assets/prefabs/furnishings/chest.prefab:3]: One drawback of this approach is that if a screen just renders this value as a text without an explicit call to `TranslationSystem#translate(...)` this unreadable string will show up...

I don't understand (yet) why using this reference string in a `.ui` file directly is automatically translated, but we return this string in a programmatic binding it is not. :thinking: 

See https://github.com/Terasology/Inventory/pull/45/files#diff-04de2f5098d10bdd839f6f8c99d30807432d03ca23c6eebe36ab5a715341109bR11

**Pull request merged**